implementations/rust/ockam/ockam_transport/DEVELOP.md

```diff
@@ -0,0 +1,136 @@
+# Develop
```

Comment:
Could you please replace DEVELOP.md and LICENSE with symlinks?

Reply:
I didn't see they were symlinks, sorry! I've also added a symlink for the `deny.toml` file.